### PR TITLE
DOCSP-42973-compass-nested-components

### PR DESCRIPTION
**Pull request by gmiller-mdb**
## DESCRIPTION

Fixed compass nested components.

## STAGING

https://deploy-preview-671--docs-compass.netlify.app/connect/advanced-connection-options/general-connection/

https://deploy-preview-671--docs-compass.netlify.app/connect/advanced-connection-options/in-use-encryption/

https://deploy-preview-671--docs-compass.netlify.app/indexes/

https://deploy-preview-671--docs-compass.netlify.app/agg-pipeline-builder/aggregation-pipeline-builder-settings/

https://deploy-preview-671--docs-compass.netlify.app/agg-pipeline-builder/export-pipeline-results/

https://deploy-preview-671--docs-compass.netlify.app/connect/advanced-connection-options/tls-ssl-connection/

https://deploy-preview-671--docs-compass.netlify.app/import-export/

## JIRA

https://jira.mongodb.org/browse/DOCSP-42973

## BUILD LOG

https://app.netlify.com/sites/docs-compass/deploys/66edc4512c859400083bdb45

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/agg-pipeline-builder/aggregation-pipeline-builder-settings.txt [hunk 1]
```diff
@@ -44,10 +44,8 @@ the upper right of the pipeline builder to open the :guilabel:`Settings` panel.
        :pipeline:`$bucket`, and :pipeline:`$bucketAuto` pipeline stages. Lower 
        limits improve pipeline run time but might result in missing documents. 
        
-       .. note:: 
-        
-          This setting is only applied to document previews. It is not applied 
-          when the pipeline is run.
+     This setting is only applied to document previews. It is not applied
```
Comment (mdb-ashley) [inline source/agg-pipeline-builder/aggregation-pipeline-builder-settings.txt:47]: This isn't rendering correctly. Could you take another look?

File: source/connect/advanced-connection-options/in-use-encryption.txt [hunk 2]
```diff
@@ -37,232 +37,230 @@ Procedure
...
+             |compass-short| does not save KMS credentials by default. Copy 
+             and save the key in an external location.
 
-            AWS
-            ~~~
+          .. _aws-compass-encryption:
 
-            You can use :v6.0:`AWS </core/queryable-encryption/fundamentals/kms-providers/#amazon-web-services-kms>` 
-            to manage your keys.
+          AWS
```
Comment (mdb-ashley) [inline source/connect/advanced-connection-options/in-use-encryption.txt:84]: For some reason, this isn't rendering. Take another look?

File: source/connect/advanced-connection-options/in-use-encryption.txt [hunk 3]
```diff
@@ -37,232 +37,230 @@ Procedure
...
+               - No 
+               - If the *Client Private Key* is protected with a password, 
+                 you must provide the password.
+
+          .. _gcp-compass-encryption:
 
-            .. list-table::
-               :header-rows: 1 
-               :widths: 30 30 70
+          GCP
```
Comment (mdb-ashley) [inline source/connect/advanced-connection-options/in-use-encryption.txt:130]: This also isn't rendering.

File: source/connect/advanced-connection-options/in-use-encryption.txt [hunk 4]
```diff
@@ -37,232 +37,230 @@ Procedure
...
+              
+             * - Client Key Password
+               - No 
+               - If the *Client Private Key* is protected with a password, 
+                 you must provide the password.
+
+
+          .. _azure-compass-encryption:
+
+          Azure
```
Comment (mdb-ashley) [inline source/connect/advanced-connection-options/in-use-encryption.txt:176]: As above.

**Review by mdb-ashley — CHANGES_REQUESTED** [review]
This is looking pretty good, Grace! I added some comments for your consideration.

File: source/connect/advanced-connection-options/in-use-encryption.txt [hunk 5]
```diff
@@ -37,232 +37,230 @@ Procedure
...
+          :v6.0:`Local KMS </core/queryable-encryption/fundamentals/kms-providers/#local-key-provider>`
+          option.  
+
+          Click :guilabel:`Generate Random Key` to generate a 96-byte long 
+          base64-encoded string. You need this key to access encrypted and 
+          decrypted data.
 
-               |compass-short| does not save KMS credentials by default. Copy 
-               and save the key in an external location.
+          .. warning::
```
[the remark this answers is not in the document]
Comment (gmiller-mdb) [reply source/connect/advanced-connection-options/in-use-encryption.txt:77]: @mdb-ashley just talked to sarah, we leave those. the only thing we change is a numbered list inside a table cell

File: source/connect/advanced-connection-options/general-connection.txt [hunk 6]
```diff
@@ -38,56 +38,53 @@ Procedure
...
 
-      .. procedure::
-         :style: connected
+      a) Select a connection string scheme.
 
-         .. step:: Select a connection string scheme.
+         You can select one of the following options:
 
-            You can select one of the following options:
+         .. list-table::
```
[the remark this answers is not in the document]
Comment (gmiller-mdb) [reply source/connect/advanced-connection-options/general-connection.txt:45]: asked sarah again, this is still fine

File: source/connect/advanced-connection-options/in-use-encryption.txt (hunk 4 above)
Comment (gmiller-mdb) [reply source/connect/advanced-connection-options/in-use-encryption.txt]: @mdb-ashley now it is actually ready for internal review!

File: source/connect/advanced-connection-options/in-use-encryption.txt [hunk 7]
```diff
@@ -61,206 +61,209 @@ Procedure
           
           - :ref:`KMIP <kmip-compass-encryption>`
 
-          .. _local-kms:
+   .. step::  Click Connect.
+
+KMS Provider Links
```
Comment (mdb-ashley) [inline source/connect/advanced-connection-options/in-use-encryption.txt:66]:
```suggestion
KMS Providers
```

File: source/connect/advanced-connection-options/in-use-encryption.txt [hunk 8]
```diff
@@ -61,206 +61,209 @@ Procedure
...
+     - Specifies the location of a local .pem file that contains 
+       either the client's TLS/SSL X.509 certificate or the client's TLS/SSL 
+       certificate and key.
+
+   * - Client Key Password
+     - No 
+     - If the *Client Private Key* is protected with a password, 
+       you must provide the password.
+
+(Optional) Specify an EncryptedFieldsMap:
```
Comment (mdb-ashley) [inline source/connect/advanced-connection-options/in-use-encryption.txt:265]:
```suggestion
(Optional) Specify an ``EncryptedFieldsMap``:
```

File: source/connect/advanced-connection-options/in-use-encryption.txt [hunk 9]
```diff
@@ -61,206 +61,209 @@ Procedure
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add an optional client-side EncryptedFieldsMap for enhanced security.
```
Comment (mdb-ashley) [inline source/connect/advanced-connection-options/in-use-encryption.txt:268]:
```suggestion
Add an optional client-side ``EncryptedFieldsMap`` for enhanced security.
```

**Review by mdb-ashley — APPROVED** [review]
LGTM % a couple of comments. Nice work on this, Grace!

**Pull request merged**